addons/purchase_stock/models/stock.py

```diff
@@ -94,6 +94,11 @@ def _compute_qty(self):
         """ Extend to add more depends values """
         return super()._compute_qty()
 
+    @api.depends('product_id.purchase_order_line_ids.product_qty', 'product_id.purchase_order_line_ids.state')
+    def _compute_qty_to_order(self):
```

Comment:
You need that? Since we are now calling the compute in write?

Reply:
I think we still need them as the compute may not always be triggered, for example these tests will fail if we remove the depends here: 
![image](https://github.com/odoo/odoo/assets/3251665/733e0b68-6f5b-4a3f-b951-8f880cc5beb1)